recipes/deepsvr/meta.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: deepsvr
+  version: "0.1.0"
+
+source:
+  url: https://github.com/griffithlab/DeepSVR/releases/download/v0.1.0/deepsvr-0.1.0.tar.gz
+  sha256: b95763c9fdfc718938ecf537a94714e2b4cbb1f083859faa7c86c2d90d22ef01 
+
+build:
+  noarch: python
+  script:
+    - flake8 . -v --config=.flake8rc
+    - nosetests --verbose
+    - python setup.py install
+  entry_points:
+    - deepsvr = deepsvr.cli:main
+
+requirements:
+  build:
+    - python==3.6.1
+    - setuptools
+    - nose
+    - flake8
+    - pandas==0.20.3
+    - numpy==1.12.1
+    - scikit-learn
+    - matplotlib
+    - seaborn
+    - bam-readcount
+    - convert_zero_one_based
+    - click
+    - tensorflow<=1.0.1
+    - keras==2.0.4
+    - h5py
+  run:
```

Comment:
Do the `run` dependencies need to be locked to these exact versions?

Reply:
Unfortunately, yes. Tensorflow is under such active development that the APIs are constantly changing. It took me a lot of time to find the exact combination of versions that would install on mac and linux on circleci. If there is a better way to do this I am open to suggestions.